packages/common/src/events/edit.ts

```diff
@@ -130,6 +131,16 @@ export async function updateHubEvent(
   return mapper.storeToEntity(model, {}) as IHubEvent;
 }
 
+export async function deleteHubEvent(
+  id: string,
+  requestOptions: IHubUserRequestOptions
+): Promise<void> {
+  await deleteEventApi({
```

Comment:
I would consider maybe adding a quick `TODO` comment here to conditionally update the `status` of the event to `removed` rather than performing a true delete when `requestOptions.params.permanentDelete` is set to `true`. We'll tackle doing that in the future if/when we support recycle bin behavior.